downloads/swan-lake-release-notes/swan-lake-2201.9.0/RELEASE_NOTE.md

```diff
@@ -463,6 +468,123 @@ public function main() returns error? {
 
 To view bug fixes, see the [GitHub milestone for Swan Lake Update 9 (2201.9.0)](https://github.com/ballerina-platform/ballerina-standard-library/issues?q=is%3Aclosed+is%3Aissue+milestone%3A%222201.9.0%22+label%3AType%2FBug).
 
+### Revamped connectors
+
+New versions of the following connectors have been released with major updates, as part of the Ballerina connector revamp initiative.
+All listed connectors have been released under new major versions, featuring significant API and functionality changes, along with improved documentation and examples.
+
+#### `asana` package
+
+- Enhanced connector APIs by incorporating resource method syntax.
+
+#### `aws.dynamodb` package
+
+- Introduced new APIs to work with database backups with DynamoDB REST APIs.
+
+#### `aws.dynamodbstreams` package
+
+- Introduced a new connector to work with DynamoDB streams.
+
+#### `aws.redshift` package
+
+- Introduced seamless connectivity to Amazon Redshift databases.
+
+#### `aws.sns` package
+
+- Enhanced the connector APIs to support all the APIs provided by the AWS SNS REST API.
+
+#### `candid` package
+
+- Introduced support for connecting to Candid's Charity Check, Essentials, and Premier REST APIs.
+
+#### `confluent.cavroserdes` package
+
+- Introduced support for serializing and deserializing data using Avro schemas stored in the Confluent Schmema Registry.
+
+#### `confluent.cregistry` package
+
+- Introduced new APIs to connect with the Confluent Schema Registry.
+
+#### `docusign.dsadmin` package
+
+- Enhanced the connector APIs by incorporating resource methods.
+- The package name has been changed to `docusign.dsadmin` from `docusign.admin`.
+
+#### `docusign.dsclick` package
+
+- Enhanced the connector APIs by incorporating resource functions.
```

Comment:
```suggestion
- Enhanced the connector APIs by incorporating resource methods.
```

Let's update wherever applicable.

Reply:
@Nuvindu FYA

Reply:
Fixed in [e7bdffd](https://github.com/ballerina-platform/ballerina-dev-website/pull/8908/commits/e7bdffd2803e9b07909b83821893089661d08ca9)